x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/use_columns.tsx

```diff
@@ -196,15 +196,15 @@ export const useRulesColumns = ({ hasPermissions }: ColumnsProps): TableColumn[]
             {value}
           </EuiText>
         ),
-        sortable: !!isInMemorySorting,
+        sortable: true,
```

Comment:
This and line 207 are only here for testing/demoing purposes. Will be removed once this PR is approved